kratos/sources/geometrical_object.cpp

```diff
@@ -0,0 +1,28 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher (https://github.com/philbucher)
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/geometrical_object.h"
+#include "includes/kratos_flags.h"
+
+namespace Kratos {
+

```

Comment:
I would move more things to the cpp to reduce compilation times

Reply:
But in a separate PR. It is important to keep the code reorganization PR separate from the otheres.